Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift

```diff
@@ -272,14 +274,18 @@ final class SubscriptionsViewController: BaseViewController {
 
     func setupTitleView() {
         if let titleView = SubscriptionsTitleView.instantiateFromNib() {
+            titleView.translatesAutoresizingMaskIntoConstraints = false
+            titleView.delegate = self
+            titleView.layoutIfNeeded()
+            titleView.sizeToFit()
+            updateServerInformation()
+
+            titleView.translatesAutoresizingMaskIntoConstraints = true
```

Comment:
@rafaelks why exactly are you setting it back to `true`?

Reply:
iOS 10 hacks 🤓

Reply:
Reference: https://stackoverflow.com/questions/15285698/building-a-titleview-programmatically-with-constraints-or-generally-constructin

Reply:
Interesting 🤔 maybe it worth it to put it as a comment above this line so everyone can easily understand and to make sure we will remember to remove it when we stop supporting iOS 10